src/ProblemDetailsResponseFactory.php

```diff
@@ -30,11 +29,11 @@
 use function print_r;
 use function sprintf;
 use function strpos;
-
 use const JSON_PRESERVE_ZERO_FRACTION;
 use const JSON_PRETTY_PRINT;
 use const JSON_UNESCAPED_SLASHES;
 use const JSON_UNESCAPED_UNICODE;
+use const JSON_PARTIAL_OUTPUT_ON_ERROR;
```

Comment:
We alphabetize imports of the same type; this entry belongs at the top of the list. :)

Reply:
Moved it to the top.